apps/src/weblab/CdoBramble.js

```diff
@@ -311,7 +376,7 @@ export default class CdoBramble {
       return;
     }
 
-    const next = () => {
+    const next = err => {
```

Comment:
Do we handle this error within 'next'?

Reply:
no, we don't, but i wanted to make it more explicit that it could be called with an error. the old behavior exited this function (calling `finalCallback`) if there was an error, but we want to write the next file if a file errors.

my future goal is to add better error-handling throughout this class, but for now i could log this error to the console to make it obvious that this needs to be handled more appropriately